client/profile-wizard/style.scss

```diff
@@ -72,9 +77,10 @@
 		margin-top: $gap-larger;
 		margin-left: auto;
 		margin-right: auto;
+		text-align: left;
 
 		> * {
-			max-width: 476px;
+			max-width: 504px;
```

Comment:
The designs have made the card bigger to compensate for the larger amount of text content.